straxen/plugins/event_processing.py

```diff
@@ -374,13 +374,11 @@ def compute(self, events):
             (0, pax_file('XENON1T_s2_xy_ly_SR0_24Feb2017.json')),
             (170118_1327, pax_file('XENON1T_s2_xy_ly_SR1_v2.2.json'))]),
    strax.Option(
-        'elife_file',
-        default=aux_repo + '3548132b55f81a43654dba5141366041e1daaf01/strax_files/elife.npy',
-        help='Electron lifetime model '
-             'To use a constant value, provide a link (as this default) To use'
-             'the corrections management tools specify the following:'
+        'elife_conf',
+        default=('elife_constant', 700000., True),
```

Comment:
Can you revert the default for 1T in the context of 1t?

Reply:
sure